app/javascript/components/search/results/StudySearchResult.jsx

```diff
@@ -170,17 +175,109 @@ function studyTypeBadge(study) {
   }
 }
 
+/**
+ * Log analytics about search result selection to Mixpanel
+ *
+ * This logging differentiates properties about the selected result from
+ * properties about other aspects of the search -- e.g. the full result set
+ * and query -- by prefixing the latter with a "namespace", i.e. `results:`.
+ *
+ * @param {Object} study Selected result
+ * @param {Object} logProps Other properties to log
+ *
+ * TODO (SCP-4256): Refine logged properties for search result selection
+ * TODO (SCP-4257): Eliminate duplicate default/custom search event logging
+ */
+export function logSelectSearchResult(study, logProps={}) {
```

Comment:
I wonder if this mixpanel-y specific function fits best in this `StudySearchResult` file or if it would make more sense to put it in the metrics file?

Reply:
Concur.  We now have a few hundred lines of search-specific analytics code, so it probably deserves it's own  file

Reply:
Agreed.  High-level reorganization makes me nervous so close to freeze, so I've added this to a downstream ticket.